bundles/core/src/main/java/com/adobe/cq/commerce/demandware/preview/PreviewServiceImpl.java

```diff
@@ -225,7 +226,7 @@ private String getPreviewContent(final String endPoint, final Resource resource,
     }
 
     private String getInstanceEndPoint() {
-        return demandwareClient.getEndpoint();
+        return clientProvider.getDefaultClient().getEndpoint();
     }
```

Comment:
we have probably two instances here.. and maybe not even a default one. this would mean, we cannot preview on a multi-instance setup.